### PR TITLE
Update references to RUM navigation

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do? What is the motivation?
DOCS-6411

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/flutter.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real
 
 ### Specify application details in the UI
 
-1. In the [Datadog app][1], navigate to **UX Monitoring** > **RUM Applications** > **New Application**.
+1. In Datadog, navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][1].
```
Comment (aliciascott) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/flutter.md:34]:
```suggestion
1. In Datadog, navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][1].
```
Same as above,  "You need the 
RUM Apps Write permission to view this page."

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/android.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ dependencies {
 
 ### Specify application details in the UI
 
-1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][2].
+1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][2].
```
Comment (aliciascott) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/android.md:60]:
```suggestion
1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][2].
```
Just an FYI I received a message "You need the
RUM Apps Write permission to view this page."
So maybe we need a Note here just to mention this ?

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/ios.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ DatadogRUM.xcframework
 
 ### Specify application details in the UI
 
-1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][5].
+1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][5].
```
Comment (aliciascott) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/ios.md:94]:
```suggestion
1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][5].
```

Same as above, "You need the
RUM Apps Write permission to view this page."

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/reactnative.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ The Datadog React Native SDK requires you to have `compileSdkVersion = 31` or hi
 
 ### Specify application details in the UI
 
-1. In the [Datadog app][1], navigate to **UX Monitoring** > **RUM Applications** > **New Application**.
+1. In Datadog, navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][1].
```
Comment (aliciascott) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/reactnative.md:75]:
```suggestion
1. In Datadog, navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][1].
```
Same as above, "You need the
RUM Apps Write permission to view this page."

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ Make sure you have a `roku_modules/datadogroku` subfolder in both the `component
 
 ### Specify application details in Datadog
 
-1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][3].
+1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][3].
```
Comment (aliciascott) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md:65]:
```suggestion
1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][3].
```
Same as above, "You need the
RUM Apps Write permission to view this page."

**Review by aliciascott — APPROVED** [review]
Hi @jhgilbert this looks great! The content is good, I just had a problem with my permissions when navigating to those Setup pages, so I think if you can document somewhere that the user needs RUM Apps Write permission to view those pages, that would help! 
FWIW it looks like the in-app links themselves didn't change, just the naming, so in the current docs we don't call this out either, will leave that up to you if you feel it's needed. Re-request review if you need to :)

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/android.md (hunk 2 above)
Comment (jhgilbert) [reply content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/android.md:60]: It should be okay for the user who would be following these steps, but I appreciate you flagging it!

**Pull request merged**